mava/utils/make_env.py

```diff
@@ -103,7 +105,14 @@ def make_jumanji_env(
     # Config generator and select the wrapper.
     generator = _jumanji_registry[env_name]["generator"]
     generator = generator(**config.env.scenario.task_config)
-    wrapper = _jumanji_registry[env_name]["wrapper"]
+
+    # If observation_type is in the config use the corresponding wrapper and make sure
+    # implicit_agent_id is set to False.
+    if hasattr(config.env, "observation_type"):
+        wrapper = _jumanji_registry[env_name]["wrapper"][config.env.observation_type]
+        config.env.implicit_agent_id = False
+    else:
+        wrapper = _jumanji_registry[env_name]["wrapper"]
```

Comment:
Not a fan of this, I think I would prefer to just make this a separate env as we discussed this morning. So all envs only ever have a single wrapper. 

If you do this we should re-use the same scenario, just have another env config and then just add that env config to the tests :pray:

Reply:
Reworked a bit to now have a separate env config for vector connector. I also added vector connector to the integration tests.